### PR TITLE
my first commit

### PR DESCRIPTION
**Pull request by kemalyilmazx**
from Kemal to dspergola -- pull request

### CODE REVIEW

File: hell_world_public_repo.ipynb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
```
Comment (dspergola) [inline hell_world_public_repo.ipynb:9]: @kemalyilmazx this is a nice piece of code!

Comment (kemalyilmazx) [reply hell_world_public_repo.ipynb:9]: Thank you Sir!

File: hell_world_public_repo.ipynb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+     "output_type": "stream",
+     "text": [
+      "Hello World!\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Hello World!\")"
```
Comment (kemalyilmazx) [inline hell_world_public_repo.ipynb:17]: How about this?

File: hell_world_public_repo.ipynb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
```
Comment (dspergola) [inline hell_world_public_repo.ipynb:22]: This is a new review!

**Pull request merged**